src/modules/apps/e2e/errorBoundary.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Color } from 'ink'
+
+export class ErrorBoundary extends React.Component<{}, { error: any }> {
+    constructor(props) {
+      super(props)
+      this.state = { error: null }
+    }
+  
+    static getDerivedStateFromError(error) {
+      // Update state so the next render will show the fallback UI.
+      return { error }
+    }
+  
+    componentDidCatch(_error, _errorInfo) {
+      // You can also log the error to an error reporting service
+      // logErrorToMyService(error, errorInfo);
+    }
```

Comment:
No need for a `componentDidCatch` here - it's used to treat errors from descendants, but there's only one that hardly will throw `Color`: https://reactjs.org/docs/react-component.html#componentdidcatch